drivers/storage/portworx/manifest/common.go

```diff
@@ -44,6 +45,9 @@ func getManifestFromURL(manifestURL string, proxy string) ([]byte, error) {
 
 		client := &http.Client{
 			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
```

Comment:
Should we take this true/false value from StorageCluster env var `DO_NOT_VERIFY_CA` instead of hardcoding? This way when we will allow user to have CA certs, we don't have to change this

Reply:
Uh.. it's not that we used `DO_NOT_VERIFY_CA` env-var that widely (only limited support in `porx` and no support in operator).  We typically use annotations for such configuration changes -- which is why I suggested the annotation.

This particular annotation could work initially for Telemetry-proxy, but eventually across the whole PX cluster.